src/views/AddProductModal.vue

```diff
@@ -12,24 +12,30 @@
   <ion-content>
     <ion-searchbar @ionFocus="selectSearchBarText($event)" v-model="queryString" :placeholder="translate('Search SKU or product name')" v-on:keyup.enter="queryString = $event.target.value; getProducts()" />
     
-    <ion-list v-for="product in products" :key="product.productId">
-      <ion-item lines="none">
-        <ion-thumbnail slot="start">
-          <ShopifyImg :src="product.mainImageUrl" />
-        </ion-thumbnail>
-        <ion-label>
-          <!-- Honouring the identifications set by the user on the settings page -->
-          <h2>{{ product[productIdentificationPref.primaryId] }}</h2>
-          <p>{{ product[productIdentificationPref.secondaryId] }}</p>
-        </ion-label>
-        <ion-icon v-if="isProductAvailableInShipment(product.productId)" color="success" :icon="checkmarkCircle" />
-        <ion-button v-else fill="outline" @click="addtoShipment(product)">{{ translate("Add to Shipment") }}</ion-button>
-      </ion-item>
-    </ion-list>
+    <template v-if="products.length">
+      <ion-list v-for="product in products" :key="product.productId">
+        <ion-item lines="none">
+          <ion-thumbnail slot="start">
+            <ShopifyImg :src="product.mainImageUrl" />
+          </ion-thumbnail>
+          <ion-label>
+            <!-- Honouring the identifications set by the user on the settings page -->
+            <h2>{{ product[productIdentificationPref.primaryId] }}</h2>
+            <p>{{ product[productIdentificationPref.secondaryId] }}</p>
+          </ion-label>
+          <ion-icon v-if="isProductAvailableInShipment(product.productId)" color="success" :icon="checkmarkCircle" />
+          <ion-button v-else fill="outline" @click="addtoShipment(product)">{{ translate("Add to Shipment") }}</ion-button>
+        </ion-item>
+      </ion-list>
 
-    <ion-infinite-scroll @ionInfinite="loadMoreProducts($event)" threshold="100px" :disabled="!isScrollable">
-      <ion-infinite-scroll-content loading-spinner="crescent" :loading-text="translate('Loading')" />
-    </ion-infinite-scroll>
+      <ion-infinite-scroll @ionInfinite="loadMoreProducts($event)" threshold="100px" :disabled="!isScrollable">
+        <ion-infinite-scroll-content loading-spinner="crescent" :loading-text="translate('Loading')" />
+      </ion-infinite-scroll>
+    </template>
+    <div v-else class="empty-state">
+      <img src="../assets/images/empty-state-add-product-modal.png" alt="empty-state" />
```

Comment:
We can use DxpImage component.

Reply:
Currently we are not able to load image of following path of assets image using the DxpImage component. Hence using img tag for now. Will update once image loading image resolves.

Created an issue for above problem: https://github.com/hotwax/dxp-components/issues/266